pkg/client/unversioned/discovery_client.go

```diff
@@ -116,21 +92,17 @@ func (d *DiscoveryClient) ServerGroups() (apiGroupList *unversioned.APIGroupList
 
 // ServerResourcesForGroupVersion returns the supported resources for a group and version.
 func (d *DiscoveryClient) ServerResourcesForGroupVersion(groupVersion string) (resources *unversioned.APIResourceList, err error) {
-	url := d.baseURL
+	url := url.URL{}
 	if groupVersion == "v1" {
```

Comment:
No. I think the plan is breaking the kube API to smaller groups and each group evolves in their own pace. We won't have a monolithic v2 API.